### ISSUE TITLE
landscape mode does not display neither top&bottom bars

### ISSUE THREAD

**Issue by kelson42** [automactic/Kiwix#38]
Tested on iphone

**Comment by automactic** [comment]
Oh, you want it? I hide it deliberately, because it is condense vertically. I can easily turn it back on.

**Comment by kelson42** [comment]
You are right, but without the bars, the app is not usable. Do you mean, each time the user want the button, he should switch to portrait?
